sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,233 @@
+ - Added a diagnostic in case of timing issues for FPGA AOT [c69a3115]
+ - Added support for C `memcpy` usages in the device code [76051ccf]
+ - [ESIMD] Added support for vectorizing scalar function [3fc66cc]
+ - Disabled vectorization and loop transformation passes because loop unrolling
+   in "SYCL optimization mode" used default heuristic, which is tuned the code
+   for CPU and might not have been profitable for other devices [ff6929e6]
+### SYCL Library
+ - Added an exception throw if no matched device is found when
+   `SYCL_DEVICE_FILTER` is set regardless of `device_selector` used [ef4e6dd]
+ - Changed event status update to complete without waiting when run on CUDA
+   devices [be7c1cb]
+ - Added `sycl::bit_cast` [d4b66bd]
+ - Improved performance when executing with dynamic batching on Level Zero
+   backend [fa382d6]
+ - Introduced pooling for USM and buffer allocations in Level Zero backend
+   [4cffedd]
+ - Added support for vectors with length of 3 and 16 elements in sub-group load
+   and store operations [4e6452d]
+ - Added interop types for images for Level Zero and OpenCL backends [a58cfef]
+ - Improved plugins discovery - continue discovering even if a plugin fails to
+   load [8c07803]
+ - Implemented queries for IEEE rounded `sqrt`/`div` in Level Zero backend
+   [91b35c4]
+ - Added SYCL 2020 `interop_handle::get_backend()` method [041ca27]
+ - [ESIMD] Deprecated `block_load`/`block_store` and
+   `simd::copy_from`/`simd::copy_to` [5c41ed6]
+ - Allowed for `const` and `volatile` pointer in sub-group `load` operation
+   [50edee4]
+ - Replaced use of `interop<>` with SYCL 2020 `backend_return_t<>` in
+   `interop_handle` [d08c21a]
+ - [ESIMD] Moved ESIMD APIs to `sycl::ext::intel::experimental::esimd` namespace
+   [92da579]
+ - Added global offset support for Level Zero backend [9ca2f911]
+ - [ESIMD] Changed `simd::replicate` API by adding suffixes into the names to
+   reflect the order of template arguments [e45408ad]
+ - Disabled persistent device code caching by default [48f6bc9e]
+ - Introduced `SYCL_REDUCTION_DETERMINISTIC` macro which forces reduction
+   algorithms to produce stable results [a3fc51a4]
+ - Improved `SYCL_DEVICE_ALLOWLIST` format [9216b49d]
+ - Added `SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING` macro to disable range
+   rounding [5c4275ac]
+ - Disabled range rounding by default when compiling for FPGA [5c4275ac]
+ - [ESIMD] Fixed a bug in `simd_view::operator--` [ccc97e23]
+ - Fixed a memory leak for host USM allocations [c18c3456]
+ - Fixed possible crashes that could happen when `sycl::free` is called while
+   there are still running kernels [c74f05d6]
+ - Deprecated `sycl::buffer::get_count()`, please use `sycl::buffer::size()`
+   instead [baf2ed9d]
+ - Implemented `sycl::group_barrier` free function [48363902]
+ - Added support of [SYCL_INTEL_enqueue_barrier extension](doc/extensions/EnqueueBarrier/enqueue_barrier.asciidoc) for CUDA backend [2e978482]
```

Comment:
New feature?

Reply:
It has been supported for a lot of time for other backends, so I'm considering it being more like a improvement that now it works for CUDA backend as well.